doc_source/resource-type-walkthrough.md

```diff
@@ -314,11 +359,14 @@ The CallbackContext is modeled as a POJO so you can define what information you
            }
 
            if (instanceStateSoFar == null) {
+               Instance instance = createEC2Instance(model);
+               model.setInstanceId(instance.getInstanceId());
```

Comment:
The instance ID needs to be set here so the primary identifier can be included in the result.